net/acme-common/files/acme.sh

```diff
@@ -8,10 +8,7 @@
 #
 # Authors: Toke Høiland-Jørgensen <toke@toke.dk>
 
-export state_dir=/etc/acme
-export account_email=
-export debug=0
-export run_dir=/var/run/acme
+run_dir=/var/run/acme
```

Comment:
Why stop exporting the run_dir? No reason the hook scripts can't use that...

Reply:
IMHO, the contract that acme-common has with acme-clients is `/var/run/acme`, not the variable `run_dir`, they shouldn't depend on a implementation detail.

Reply:
Actually, I'm not sure about this. It's mostly used to expose `/var/run/acme/challenge`. What do you think, should we expose `/var/run/acme/challenge` or `$run_dir/challenge`?